### PR TITLE
Update supported platforms page to be more user friendly

### LINKED ISSUE #118: Update content of supported_platforms page
Currently https://adoptium.net/supported_platforms.html states "We're currently working out our list of supported platforms. Please check back later!" (see https://github.com/adoptium/adoptium.net/blob/main/src/handlebars/supported_platforms.handlebars#L11).  

We have a draft set of platforms listed in the 2021 program plan.  This issue is to get that finalized and shared (via this website page).

### PR DESCRIPTION
**Pull request by gdams**
##### Checklist

- [x] `npm test` passes
- [ ] documentation is changed or added (if applicable)
- [ ] permission has been obtained to add new logo (if applicable)
- [x] contribution guidelines followed [here](https://github.com/adoptium/adoptium.net/blob/master/CONTRIBUTING.md)

closes: https://github.com/adoptium/adoptium.net/issues/118

### CODE REVIEW

File: src/handlebars/supported_platforms.handlebars [hunk 1]
```diff
@@ -5,1055 +5,271 @@
...
-            <tbody>
-                <tr>
-                    <td>OSX 10.10+</td>
-                    <td><i class="fa fa-star-o hotspot" aria-hidden="true"></i><span class="sr-only">Hotspot VM</span></td>
-                </tr>
-            </tbody>
-        </table>
-
-    </div>
+  <h3>This section lists the operating systems that are supported with the latest release of Eclipse Temurin.</h3>
```
Comment (karianna) [inline src/handlebars/supported_platforms.handlebars:8]: indenting please

**Review by karianna — CHANGES_REQUESTED** [review]
whitespace changes otherwise good.

File: src/handlebars/supported_platforms.handlebars (hunk 1 above)
Comment (gdams) [reply src/handlebars/supported_platforms.handlebars:8]: just pushed a commit

**Comment by gdams** [comment]
Dumping my thoughts here so I don't forget.

Centos is no longer on s390x (it's ClefOS)

We should likely support SLES on s390x

**Review by karianna — APPROVED** [review]
LGTM

**Comment by gdams** [comment]
**Suggestions from PMC**

centos/rhel check on arm32

remove centos7 arm64

remove ClefOS for s390x

add glibc and macos 10.12 footnote

add Solaris update version (10u11)

Alpine Linux (clarify that it's headless)

**Review by sxa — CHANGES_REQUESTED** [review]
Can we update the footnotes to include the `libc` versions for the other Linux versions as well i.e. 2.23 on arm32, 2.17 elsewhere. Although since the Linux/arm32 one presumably works on CentOS7 we could just specify 2.17 for all non-x64 ones, but I'm a little surprised it runs ok :-) We do no testing on that.

**Comment by gdams** [comment]
> Can we update the footnotes to include the libc versions for the other Linux versions as well i.e. 2.23 on arm32, 2.17 elsewhere. Although since the Linux/arm32 one presumably works on CentOS7 we could just specify 2.17 for all non-x64 ones, but I'm a little surprised it runs ok :-) We do no testing on that.

Updated

**Review by sxa — APPROVED** [review]
LGTM now. We should definitely look to add SLES back in at least for Linux/s390x when we get a machine stood up for that, but the current table is good and looks vastly better than the old one. Thanks.

**Pull request merged**